images/linux/scripts/installers/mysql.sh

```diff
@@ -12,6 +12,9 @@ export ACCEPT_EULA=Y
 # Install MySQL Client
 apt-get install mysql-client -y
 
+# InstallMySQL database development files
+apt-get install libmysqlclient-dev
```

Comment:
should `-y` be specified? I'm guessing yes...

Reply:
yeah, let's add it just to be on the safe side